docs/proposals/003-model-server-protocol/README.md

```diff
@@ -24,13 +24,13 @@ Note the requirements here are aligned with the
 [model server metrics standardization](https://docs.google.com/document/d/1SpSp1E6moa4HSrJnS4x3NpLuj88sMXr2tbofKlzTZpk)
 effort.
 
-The corresponding metrics in vLLM are also shown in the table below, as vLLM is already integrated
-into the reference endpoint picker implementation.
 
 | Metric | Type | Description | vLLM metric | Triton TensorRT-LLM| SGLang |
-| ----- | ---- | ---- | ---- | ---- | ---- |
+| ----- | ---- | ------------ | ---- | ---- | ---- |
 | TotalQueuedRequests         | Gauge     | The current total number of requests in the queue.| `vllm:num_requests_waiting`| `nv_trt_llm_request_metrics{request_type=waiting}`| `sglang:num_queue_reqs`
 | KVCacheUtilization| Gauge     | The current KV cache utilization in percentage.| `vllm:gpu_cache_usage_perc`| `nv_trt_llm_kv_cache_block_metrics{kv_cache_block_type=fraction}`| `sglang:token_usage`
+| [Optional] BlockSize         | Labeled     | The block size in tokens to allocate memory. If this metrics is not available, it can be configured via the [prefix plugin config](https://gateway-api-inference-extension.sigs.k8s.io/guides/epp-configuration/prefix-aware/#customize-the-prefix-cache-plugin).| name: `vllm:cache_config_info`, label name: `block_size`| | 
```

Comment:
typo? `If this metric is available`; same thing below

Reply:
this was saying if the metrics is not available the info can be configured via the plugin config. Rephrased to avoid the confusion.